src/olympia/addons/models.py

```diff
@@ -237,57 +238,137 @@ def not_disabled_by_mozilla(self):
     def get_base_queryset_for_queue(
         self,
         admin_reviewer=False,
-        admin_content_review=False,
-        show_pending_rejection=False,
-        listed=True,
+        content_review=False,
+        theme_review=False,
+        exclude_listed_pending_rejection=True,
+        select_related_fields_for_listed=True,
+        select_related=True,
```

Comment:
`select_related` is unused?

Reply:
Ah yes, I refactored this method a few times and forgot to remove that parameter